internal/test.sh

```diff
@@ -7,14 +7,15 @@ if [[ $# -gt 0 ]]; then
 fi
 
 # ensure_go_binary verifies that a binary exists in $PATH corresponding to the
-# given go-gettable URI. If no such binary exists, it is fetched via `go get`.
+# given go-gettable URI. If no such binary exists, it is fetched via `go install`
+# at latest.
 ensure_go_binary() {
   local binary=$(basename $1)
   if ! [ -x "$(command -v $binary)" ]; then
-    info "Installing: $1"
+    echo "Installing: $1"
     # Run in a subshell for convenience, so that we don't have to worry about
     # our PWD.
-    (set -x; cd && env GO111MODULE=on go get -u $1)
+    (set -x; cd && env GO111MODULE=on go install $1@latest)
```

Comment:
I think just `go install $1@latest` is sufficient - GO111MODULE is default now.

(optional) it seems that we just need staticcheck test in addition to go test ./... and this shell isn't very useful.
We can consider to replace staticcheck with https://staticcheck.io/docs/running-staticcheck/ci/github-actions/ or clone of it, and get rid of this bash script completely and make this workflow work on windows too.

Reply:
Agreed. I'll send a follow up for updating the staticcheck #78